ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 
-echo "liquid core was born ready"
+set -e
+set -x
+
+cd /opt/liquid-core/liquid-core
+../venv/bin/python ./manage.py migrate
+../venv/bin/python ./manage.py collectstatic --noinput
```

Comment:
Eh, this should actually be done during setup, those static files aren't going to change.

Reply:
Right, committing fix now